common/build-helper/texmf.sh

```diff
@@ -0,0 +1,25 @@
+# Avoid repeating same thing accross a gazillion files
+
+# rsync isn't needed for everything but it's far easier to just put it here
+hostmakedepends+=" unzip xz rsync"
+# python_version isn't needed for everything either
+python_version=3
+
+do_install() {
+	mkdir -p "${PKGDESTDIR}/usr/share/texmf-dist"
+	unzip "${pkgname}-${version}-src.zip"
+	find . -maxdepth 1 -print -name "*.tar.xz" \
```

Comment:
Really only xz?

Reply:
Yes, inside the zip are a bunch of .tar.xz files...